mongodb-enterprise-openshift.yaml

```diff
@@ -152,20 +142,41 @@ rules:
     verbs:
       - get
 
+
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: mongodb-enterprise-appdb
-  namespace: mongodb
+
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: mongodb-enterprise-appdb
 subjects:
   - kind: ServiceAccount
     name: mongodb-enterprise-appdb
-    namespace: mongodb
+
+
+---
+# Source: mongodb-enterprise-operator/templates/serviceaccount.yaml
```

Comment:
Good idea with `imagePullSecrets` for service accounts
though the `mongodb-enterprise-openshift.yaml` already contains instructions for these two service accounts